AudioDeviceExample/ViewController.swift

```diff
@@ -79,6 +154,10 @@ class ViewController: UIViewController {
                 builder.videoTracks = [videoTrack]
             }
 
+            if let audioTrack = self.localAudioTrack {
+                builder.audioTracks = [audioTrack]
+            }
```

Comment:
The comment above is out of date, and should probably be moved directly above 157.

Reply:
Is it worth it to mention that we will only share a LocalAudioTrack if `ExampleAVAudioEngineDevice` is selected?